### ISSUE TITLE
Only works on own repositories

### ISSUE THREAD

**Issue by evfool** [sourcerer-io/sourcerer-app#228]
The idea is great, I always wanted to have a complete profile of my open-source contributions.
Unfortunately there's no way of marking repositories which I am not the owner of, and I am not part of the organization maintaining it, to be crawled for commits. Would it be possible to also crawl e.g. starred repositories for the users' contributions to be shown on the profile?

Example https://github.com/evfool is my profile
Example repository not shown on sourcerer profile: https://github.com/GNOME/gnome-system-monitor (or several other projects I've contributed to under the GNOME org umbrella, where these are just github mirrors or git repositories hosted elsewhere) is a repository with several contributions done by my user, showing my user icon as a reference.
Example commits associated to my user, but not visible on the sourcerer.io profile: https://github.com/GNOME/gnome-system-monitor/commits?author=evfool (the association happens via an email address set on my github profile, not the primary one though)

**Comment by anatolykislov** [comment]
Hi @evfool !
Thank you for a feedback! We are very glad that you like our product.
We are planning to add the ability to specify repo URLs in the user's settings menu. With it, you will be able to add gnome-system-monitor repo manually. How do you think this is a good solution?

**Comment by evfool** [comment]
Yes, adding repos manually in the user settings would also a perfect solution.

**Comment by shawncatz** [comment]
I'd actually like to see the app discover the organizations that I'm a part of and let me choose which ones to add, or something like that. I'm part of 25+ organizations on github and work on over 100 repos regularly :)

**Comment by anatolystansler** [comment]
Hi @shawncatz! It can be done already on sign up. But we can't discover organizations which is not publicly listed in your GitHub profile.

Adding of a repo by URL is WIP.

**Comment by shawncatz** [comment]
By sign up, I'm guessing you mean new user flow? Is there a way to redo that?

**Comment by anatolystansler** [comment]
@shawncatz you can edit organizations permissions at our [GitHub application page](https://github.com/settings/connections/applications/234472f42264e3374002) and then in your profile click "Update -> Update GitHub repos".

**Comment by shawncatz** [comment]
ahh cool, good to know. Thanks :)

**Comment by adrianoavelino** [comment]
I was looking for that. Very thanks for helping us with that
